src/pages/jonghoLee/MainPage/Main/MainLeft/Comment.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import './Comment.scss';
+
+import smileImg from '../../../../../images/jonghoLee/smile.png';
+
+export default class Comment extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      comment: "",
+      isDisabled: true,
+    }
+  }
+
+  commentBtnClickHandler = (e) => {
+    e.preventDefault();
+    this.makeComment();
+  }
+
+  textKeyPressHandler = (e) => {
+    if (e.key === 'Enter' && this.state.comment.trim().length > 0) {
+      this.makeComment();
+    }
+  }
+
+  makeComment = () => {
+    this.props.createComment(this.state.comment);
+    this.setState({
+      comment: '',
+      isDisabled: true
+    })
+  }
+
+  textChangeHandler = (e) => {
+    this.setState({
+      comment: e.target.value,
+      isDisabled: (e.target.value.trim().length > 0) ? false : true,
+    });
+  }
```

Comment:
```suggestion
  textChangeHandler = (e) => {
    const isDisabled = !(e.target.value.trim().length > 0);
    
    this.setState({
      comment: e.target.value,
      isDisabled
    });
  }
```
- boolean 데이터를 좀 더 활용해볼 수 있겠네요!